ui/src/shared/components/InputClickToEdit.tsx

```diff
@@ -130,13 +132,15 @@ class InputClickToEdit extends PureComponent<Props, State> {
             onFocus={this.handleFocus}
             tabIndex={tabIndex}
             spellCheck={false}
+            data-test={`${testId}--input`}
```

Comment:
nit: this generates `undefined--input` on unspecified testId ... it should be better be undefined